meterfs/meterfs.c

```diff
@@ -1375,6 +1416,32 @@ int meterfs_devctl(const meterfs_i_devctl_t *i, meterfs_o_devctl_t *o, meterfs_c
 			ctx->earlyErase = i->setEarlyErase.enable;
 			break;
 
+		case meterfs_reset:
+			p = node_getById(i->id, &ctx->nodesTree);
+			if (p == NULL) {
+				return -ENOENT;
+			}
+
+			err = meterfs_resetFile(p, ctx);
+			if (err < 0) {
+				return err;
```

Comment:
Perhaps `break` instead, disputable

Reply:
Fine by me, but other cases are mostly exiting by return, should I rewrite them too? (e.g. as an additional commit)